hack/generators/generate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+export PATH=$PATH:$(pwd)/hack/generators
+pwd
+for file in $(find . -name generate.json); do
```

Comment:
`file` is a standard system command. I'm glad this isn't broken but maybe consider using a different name instead.